test/utils/cryptography/P256.t.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {P256} from "@openzeppelin/contracts/utils/cryptography/P256.sol";
+import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
+
+contract P256Test is Test {
+    /// forge-config: default.fuzz.runs = 512
```

Comment:
any reason for 512 runs? is pretty low imo, especially as the workflow will use this value instead of 5000 from foundry.toml

Reply:
It gets very long very fast. 500 is already pretty good IMO